frontend/src/pages/Recipe.js

```diff
@@ -0,0 +1,281 @@
+            >
+              "{recipe?.response?.description}"
+            </Typography>
+          </ThemeProvider>
+          <Box
+            sx={{
+              display: "flex",
+              alignItems: "center",
+              justifyContent: {
+                xs: "space-around",
+                md: "center", 
+              },
+              gap: {
+                md: "30px", 
+              },
+              paddingBottom: {
+                md: "0", 
+              },
+              marginBottom: {
+                xs: "30px", 
+                md: "0", 
+              },
+            }}
+          >
+            <Box
+              style={{
+                display: "flex",
+                alignItems: "center",
+                color: "#DB398D",
+              }}
+            >
+              <AccessTimeIcon color="primary" style={{ marginRight: "5px" }} />{" "}
+              {recipe?.response?.recipeCookingTime}
+            </Box>
+            <Box
+              style={{
+                display: "flex",
+                alignItems: "center",
+                color: "#DB398D",
+              }}
+            >
+              <MenuBookIcon color="primary" style={{ marginRight: "5px" }} />{" "}
+              {recipe?.response?.recipeCategory}
+            </Box>
+          </Box>
+        </Box>
+        <Box
+          sx={{
+            paddingBottom: {
+              md: "100px", 
+            },
+            display: {
+              md: "grid", 
+            },
+            gridTemplateColumns: {
+              md: "1fr 2fr", 
+            },
+            gap: {
+              md: "30px", 
+            },
+          }}
+        >
+          <Box
+            sx={{
+              bgcolor: grey[100],
+              float: {
+                md: "left", 
+              },
+              padding: {
+                xs: "15px 20px 5px", 
+                md: "30px 35px 35px", 
+              },
+              borderBottomRightRadius: {
+                xs: "20px", 
+                md: "20px", 
+              },
+            }}
+          >
+            <h2>Ingredienser</h2>
+            <div>
+              {recipe?.response?.recipeIngredients.map(
+                (recipeIngredients, index) => {
+                  return (
+                    <p key={index}>
+                      {recipeIngredients.recipeIngredientAmount}{" "}
+                      {recipeIngredients.recipeIngredientUnit}{" "}
+                      {recipeIngredients.recipeIngredient}
+                    </p>
+                  );
+                }
+              )}
+            </div>
+          </Box>
+          <Box
+            sx={{
+              paddingTop: {
+                md: "30px", 
+              },
+            }}
+          >
+            <h2>Gör så här</h2>
+            <div>
+              {recipe?.response?.recipeInstruction?.map(
+                (recipeInstruction, index) => {
+                  return (
+                    <li
```

Comment:
According to HTML docs, `<li />` element can only be placed inside `<ul />`, but in your case it's a child of` <div />` - it definitely should be changed